modules/bluefish/CMakeLists.txt

```diff
@@ -3,18 +3,18 @@ project (bluefish)
 
 set(SOURCES
 		consumer/bluefish_consumer.cpp
-
 		util/blue_velvet.cpp
-
 		bluefish.cpp
 		StdAfx.cpp
 )
 set(HEADERS
+		interop/BlueDriver_p.h
+		interop/BlueHancUtils.h
+		interop/BlueVelvetC.h
+		interop/BlueVelvetCUtils.h
```

Comment:
I think these should still have been under dependencies64/bluefish/include as before since they are part of your SDK not really part of CasparCG. What do you think?